load-test-framework/src/main/java/com/google/pubsub/flic/Driver.java

```diff
@@ -189,6 +202,10 @@ private void run() {
       Client.maxOutstandingRequests = maxOutstandingRequests;
       Client.burnInTimeMillis = (Client.startTime.getSeconds() + burnInDurationSeconds) * 1000;
       Client.numberOfMessages = numberOfMessages;
+      Client.cpsPublisherCount = cpsPublisherCount;
```

Comment:
That's correct, it isn't the static part I'm objecting to as much as the part it's part of the Client class. It could be part of the Controller / GCEController class.